lib/blackout_updater.rb

```diff
@@ -0,0 +1,37 @@
+class BlackoutUpdater
+  # Service Object to update blackouts in the blackouts controller
+  def initialize(blackout:, update_p:)
```

Comment:
`update_p` should be renamed to either `params`, `new_attrs`, or `update_params`, take your pick!